src/intent.js

```diff
@@ -0,0 +1,9 @@
+function intent(DOM) {
+  let actions = {
+    userScrolled$: DOM.get('#scroll-table-container', 'scroll')
+      .map(e => e.srcElement.scrollTop)
+  };
+  return actions;
+}
+
+export default intent;
```

Comment:
The only reason why this `userScrolled$` is defined here and not in the model is because you don't want your model to have anything directly related to the DOM or to user interaction.